### PR TITLE
install redpanda in docker entrypoint

### PR DESCRIPTION
**Pull request by andrewhsu**
This PR includes the changes in PR #3 

This PR refactors the way redpanda is installed in the docker containers. Instead of installing redpanda when the image is built, it is installed when the container is started via the `entrypoint.sh` script. Still expects the `DEB_PATH` environment variable as the path to the redpanda debian package outside of the container.

This change allows the docker image for chaos testing to not be strongly tied to a specific redpanda version.

### CODE REVIEW

File: docker/redpanda/entrypoint.sh [hunk 1]
```diff
@@ -1,5 +1,14 @@
 #!/bin/bash
 
+if [[ ! -r /mnt/vectorized/redpanda.deb ]]; then
+  echo 'error: unable to read /mnt/vectorized/redpanda.deb'
+  exit 1
+fi
+
+dpkg --force-confold -i /mnt/vectorized/deb/redpanda.deb
```
Comment (rystsov) [inline docker/redpanda/entrypoint.sh]: there is a typo, should be

`dpkg --force-confold -i /mnt/vectorized/redpanda.deb`

not

`dpkg --force-confold -i /mnt/vectorized/deb/redpanda.deb`

File: docker/redpanda/entrypoint.sh [hunk 2]
```diff
@@ -1,5 +1,14 @@
 #!/bin/bash
 
+if [[ ! -r /mnt/vectorized/redpanda.deb ]]; then
```
Comment (rystsov) [inline docker/redpanda/entrypoint.sh:5]: can you add `set -e` to prevent errors?

**Review by rystsov — COMMENTED** [review]
There is a typo but otherwise looks good!

File: docker/redpanda/entrypoint.sh (hunk 1 above)
Comment (andrewhsu) [reply docker/redpanda/entrypoint.sh]: ah, nice catch. i'll fix.

File: docker/redpanda/entrypoint.sh (hunk 2 above)
Comment (andrewhsu) [reply docker/redpanda/entrypoint.sh:5]: added with 0d6a17c

File: docker/redpanda/entrypoint.sh (hunk 1 above)
Comment (andrewhsu) [reply docker/redpanda/entrypoint.sh]: fixed with force push of 15f3931

**Review by rystsov — APPROVED** [review]

**Pull request merged**